crypto/keyring/signing_algorithms_test.go

```diff
@@ -51,25 +51,25 @@ func TestAltSigningAlgoList_Contains(t *testing.T) {
 	list := SigningAlgoList{hd.Secp256k1}
 
 	require.True(t, list.Contains(hd.Secp256k1))
-	require.False(t, list.Contains(notSupportedAlgo{}))
+	require.False(t, list.Contains(NotSupportedAlgo{}))
 }
 
 func TestAltSigningAlgoList_String(t *testing.T) {
-	list := SigningAlgoList{hd.Secp256k1, notSupportedAlgo{}}
+	list := SigningAlgoList{hd.Secp256k1, NotSupportedAlgo{}}
 	require.Equal(t, fmt.Sprintf("%s,notSupported", string(hd.Secp256k1Type)), list.String())
 }
 
-type notSupportedAlgo struct {
+type NotSupportedAlgo struct {
```

Comment:
why do we need this change?

Reply:
It is used in this [test](https://github.com/cosmos/cosmos-sdk/blob/cyberbono3/keyring-migration/crypto/keyring/keyring_ledger_test.go#L105)